### PR TITLE
Feature/small refactor

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: app/controllers/Auth/LoginController.js [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+     *
+     *  @apiErrorExample Error-Response:
+     *     HTTP/1.1 401 Unauthorized
+     *     {
+     *       "auth": false,
+     *       "token": null,
+     *        "user": null
+     *     }
+     */
+    async invoke(request, response, next) {
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/Auth/LoginController.js]: is this `next` necessary?

File: app/controllers/ToDo/DestroyController.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            if (!todo) {
+                return response.sendStatus(StatusCodes.NOT_FOUND);
+            }
+
+            if (todo.userId !== request.loggedUserId) {
+                return response.sendStatus(StatusCodes.FORBIDDEN);
+            }
+
+            todo.destroy()
+                .then(() => {
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/ToDo/DestroyController.js:34]: please change `then-catch` to `async/await` in next PR

File: app/controllers/Auth/LoginController.js (hunk 1 above)
Comment (piotr-major-rmtcfm-com) [reply app/controllers/Auth/LoginController.js:44]: now yes, it doesnt work without this

File: app/controllers/ToDo/DestroyController.js (hunk 2 above)
Comment (piotr-major-rmtcfm-com) [reply app/controllers/ToDo/DestroyController.js:34]: this is what i want to do

File: app/controllers/Users/DestroyController.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const { StatusCodes } = require('http-status-codes');
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/controllers/Users/DestroyController.js]: change folder name to `User` without `s`

File: app/index.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ const router = require('./routes');
 const ToggleAPIDocs = require('./middleware/ToggleAPIDocs');
 
 const config = require('../config');
+const errorHandler = require('./plugins/errorHandler');
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/index.js:15]: this needs also `wrapper` for correctly work, but in next PR

File: app/models/PasswordReset.js [hunk 5]
```diff
@@ -1,75 +1,78 @@
...
+
+            return token;
+        }
+
+        generateToken() {
+            return crypto.randomBytes(64).toString('hex');
+        }
+
+        async getByToken(token) {
+            return await PasswordReset.findOne({
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/models/PasswordReset.js]: you can remove async/await

File: app/models/PasswordReset.js [hunk 6]
```diff
@@ -1,75 +1,78 @@
...
                 allowNull: false
             },
-            valid_until: {
+            validUntil: {
                 type: DataTypes.DATE,
                 allowNull: false
             }
         },
-        {underscored: true}
+        { sequelize, tableName: 'PasswordResets' }
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/models/PasswordReset.js]: `tableName` is not necessary I think

File: app/models/Todo.js [hunk 7]
```diff
@@ -31,15 +46,11 @@ module.exports = (sequelize, DataTypes) => {
             }
         },
         {
-            underscored: true,
+            sequelize,
+            modelName: 'Todo',
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/models/Todo.js]: modelName and tableName not needed I think, if there are regular names

File: app/models/User.js [hunk 8]
```diff
@@ -70,9 +92,10 @@ module.exports = (sequelize, DataTypes) => {
             }
         },
         {
-            underscored: true,
+            sequelize,
+            tableName: 'Users',
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/models/User.js]: as above

**Pull request merged**